src/pages/App.tsx

```diff
@@ -83,27 +87,52 @@ export class App extends React.Component<Properties, State>  {
 
   getMasthead(loc: TranslationFunction): MastheadProperties {
     return {
-      branding: loc('masthead'),
-      user: {
-        displayName: 'John Smith',
-        email: 'jsmith@example.com',
-        menuExpanded: this.state.isUserMenuExpanded,
-        onMenuClick: this.handleUserMenuToggle,
-        menuItems: [
-          {
-            key: 'toggle-theme',
-            label: 'Toggle theme',
-            onClick: this.handleThemeToggle
-          }
-        ]
-      },
-      attr: {
-        userMenuAriaLabel: 'User Menu',
-        mobileMenuAriaLabel: 'Application Menu'
+      branding: (
+        <>
```

Comment:
what's it?

Reply:
It's a shorthand for React.Fragment, which allows you to return multiple children without wrapping in an array: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html